src/assets/app.extensions.json

```diff
@@ -1358,7 +1358,7 @@
           "id": "app.libraries.thumbnail",
           "key": "$thumbnail",
           "type": "image",
-          "class": "image-table-cell",
+          "class": "image-table-cell custom-image-cell",
```

Comment:
can we rename it to something meaningful, like "library-image-cell" ? otherwise nobody will remember what "custom" stands for in a release or two :)